### DISCUSSION TITLE
Cached Updates

### DISCUSSION THREAD

**Discussion by DJBenson** [Hyundai-Kia-Connect/kia_uvo#658] [General]
I have a couple of questions around cached updates;

- Do they count towards API limits?
- What kind of update does the official app do? On opening the app, is that a cached update and on clicking the refresh button that's a forced update?
- I understand cached updates come from Kia's server rather than polling the car but what triggers the car to update the cache? Is it periodic, does specific actions cause an update?

**Comment by fuatakgun** [comment]
1. we do not know.
2. yes
3. for my own experience, charging the car is updating data on the cloud but this is triggered by the car itself. That is because, we have two different schedule and you can block force updates at night assuming that you are charging it already.
